content/docs/command-reference/plots/modify.md

```diff
@@ -123,6 +123,29 @@ plots:
       title: Accuracy
 ```
 
+## Example: Make a regular output into a plot
+
+We can use `dvc plots modify` to make an <abbr>output</abbr> or metrics file of
+a stage into a plot. Lets take an example stage `evaluate` which has an
+<abbr>output</abbr> `logs.csv`:
```

Comment:
Names before nouns:
```suggestion
a stage into a plot. Lets take an example `evaluate` stage, which has an
`logs.csv` output:
```

Reply:
(also, note there's no need for 2 tooltips of the same term in the same paragraph, again, please always look at your work in browser to detect these issues)